### PR TITLE
Add notes from the 2022-12 meeting

### PR DESCRIPTION
**Pull request by encukou**

### CODE REVIEW

File: docs/monthly-meeting/2022-12.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- CAM / `@CAM-Gerlach`
+- Jim DeLaHunt / `@JDLH`
+
+
+## Reports and celebrations
+
+> This is a place to make announcements (without a need for discussion). This is also a great place to give shout-outs to contributors! We'll read through these at the beginning of the meeting.
+
+- Hugo's promotion to Core Developer! 🎊
+- @JDLH: A [packaging] docs issue I filed has been fixed, I feel like I made a (small) contribution. [*No clear list of permitted keys in "Declaring project metadata" specification* issue #1161](https://github.com/pypa/packaging.python.org/issues/1101), fixed by Brett Cannon in [PR #1168](https://github.com/pypa/packaging.python.org/pull/1168).
```
Comment (ezio-melotti) [inline docs/monthly-meeting/2022-12.md:30]:
```suggestion
- @JDLH: A [packaging] docs issue I filed has been fixed, I feel like I made a (small) contribution. [*No clear list of permitted keys in "Declaring project metadata" specification* issue #1101](https://github.com/pypa/packaging.python.org/issues/1101), fixed by Brett Cannon in [PR #1168](https://github.com/pypa/packaging.python.org/pull/1168).
```

File: docs/monthly-meeting/2022-12.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  * @CAM-Gerlach will open issue proposing a list of obselete PEPs to the SC to be formally rejected
+  * We may need an obselete status?
+
+### Improve SEO for CPython docs
+
+* (Hugo) OpenGraph metadata PRs:
+
+  * devguide: [merged](https://github.com/python/devguide/pull/953), increases Google's SEO score [91->100](https://github.com/python/devguide/pull/953#issuecomment-1332183148)
+  * CPython: [ready for review](https://github.com/python/cpython/pull/99931), increases Google's SEO score [~66->91](https://github.com/python/cpython/pull/99931) -> approved during and merged after the meeting :rocket:
+  * Followup: tell Marc-André about use of logo without font
```
Comment (ezio-melotti) [inline docs/monthly-meeting/2022-12.md:74]: This is now done, we removed the text and left the logo, so the font is no longer a problem.

File: docs/monthly-meeting/2022-12.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+* Final PEPs and the canonical docs banner link
+  * [python/peps#2719](https://github.com/python/peps/issues/2719) will improve the existing banners directing users to the canonical docs for Final PEPs added in [python/peps#2702](https://github.com/python/peps/pull/2702) to be sticky at the top; @CAM-Gerlach to submit PR
+  * We can start going through Final PEPs and systemically marking them with the new banner as appropriate
+  * @CAM-Gerlach will submit PRs for [python/peps#2872](https://github.com/python/peps/issues/2872) to mark old implemented Accepted PEPs as Final, and will add banner in the process
+  * We might need a checklist for marking PEPs Final
+    * e.g.: is it documented outside the PEP? Add a note?
+    * @CAM-Gerlach will open an issue to discuss adding checklists for new Approved/Rejected and Final PEPs
```
Comment (CAM-Gerlach) [inline docs/monthly-meeting/2022-12.md:63]:
```suggestion
    * @CAM-Gerlach opened issue [python/peps#2937](https://github.com/python/peps/issues/2937) to discuss adding checklists for new Approved/Rejected and Final PEPs
```

**Review by CAM-Gerlach — COMMENTED** [review]
Two tweaks so far...

File: docs/monthly-meeting/2022-12.md (hunk 3 above)
Comment (hugovk) [reply docs/monthly-meeting/2022-12.md:63]: Generally, meeting minutes should record what happened during the meeting, so putting this action point in past tense makes it sound like it had already been done before the meeting.

Maybe keep it future tense, and add a little followup note to say it's now been done?

Comment (CAM-Gerlach) [reply docs/monthly-meeting/2022-12.md:63]: Right, but I saw that you added a bunch of things after so... :stuck_out_tongue_winking_eye: 

In any case, I removed it inline and added it in a paren.

**Comment by encukou** [comment]
Thanks for the review!

**Pull request merged**